vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -116,29 +173,35 @@ $( document ).ready(function() {
   Modified on <span class="date">${space.modificationDate}</span> by ${space.modifiedBy}.     
 </div>
 
-<div id="createSpaceLinkAlert" class="alert alert-secondary" role="alert" style="cursor:move; width:250px; height: 400px; display:none; position: absolute; top: 100px; right: 50px; z-index:999">
-  <h6 class="alert-heading"><small>Create new Space Link</small></h6>
-  <p><small>Please click on the image where you want to place the new space link. Then click "Create Space Link".</small></p>
-  <hr>
-  <label style="margin-right: 5px;"><small>Rotation:</small> </label>
-  <input class="form-control-xs" type="number" id="spaceLinkRotation" value="0"><br>
-  
-  <label style="margin-right: 5px;"><small>Type:</small> </label>
-  <select id="type" class="form-control-xs">
-  	<option selected value="">Choose...</option>
-  	<option value="ALERT">Alert</option>
-  </select><br>
-  
-  <label style="margin-right: 5px;"><small>Linked Space:</small> </label>
-  <select id="linkedSpace" class="form-control-xs">
-        <option selected value="">Choose...</option>
-        <c:forEach items="${spaces}" var="space">
-        <option value="${space.id}">${space.name}</option>
-        </c:forEach>
-  </select>
-  <HR>
-  <p class="mb-0 text-right"><button id="cancelSpaceLinkBtn" type="button" class="btn btn-light btn-xs">Cancel</button> <button id="createSpaceLinkBtn" type="button" class="btn btn-primary btn-xs">Create Space Link</button></p>
-</div>
+<form>
```

Comment:
is there a reason you wrapped this into a form?

Reply:
I have changed the type of Cancel button to 'RESET' so that all values will be cleared from the fields upon clicking the Cancel button which needed the whole div to be a part of a form.

Reply:
ah, that makes sense